flow360/component/simulation/translator/solver_translator.py

```diff
@@ -87,13 +87,13 @@ def get_solver_json(
             for surface in model.entities.stored_entities:
                 spec = dump_dict(model)
                 spec.pop("surfaces")
-            if isinstance(model, Wall):
-                spec.pop("useWallFunction")
-                spec["type"] = "WallFunction" if model.use_wall_function else "NoSlipWall"
-                if model.heat_spec:
-                    spec.pop("heat_spec")
-                    # TODO: implement
-            translated["boundaries"][surface.name] = spec
+                if isinstance(model, Wall):
```

Comment:
Ok, so I don't know what is wrong because I am getting only one entry for boundary. And you can see that last line of this change is using last `surface.name`

Reply:
You can check my latest commit, I think now is correct